discovery.go

```diff
@@ -212,31 +212,48 @@ func (r *Router) DiscoveryAllBuckets(ctx context.Context) error {
 	return nil
 }
 
-// startCronDiscovery is discovery_service_f analog with goroutines instead fibers
-func (r *Router) startCronDiscovery(ctx context.Context) error {
+// cronDiscovery is discovery_service_f analog with goroutines instead fibers
+func (r *Router) cronDiscovery(ctx context.Context) {
+	var iteration uint64
+
 	for {
 		select {
 		case <-ctx.Done():
 			r.metrics().CronDiscoveryEvent(false, 0, "ctx-cancel")
-
-			return ctx.Err()
+			r.log().Infof(ctx, "cron discovery has been stopped after %d iterations", iteration)
```

Comment:
Add the [DISCOVERY] tag at the beginning of the discovery log

Reply:
Done